data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml

```diff
@@ -1,63 +1,64 @@
 id: ocd-organization/22fcc747-bc8d-4a3f-b610-65c766fa1beb
 jurisdiction: ocd-jurisdiction/country:us/state:tx/government
 classification: committee
-name: Educational Opportunity & Enrichment, Select
+name: Educational Opportunity and Enrichment Select
 chamber: lower
 sources:
 - url: https://capitol.texas.gov/Committees/CommitteesMbrs.aspx?Chamber=H
 - url: https://capitol.texas.gov/Committees/MembershipCmte.aspx?LegSess=88R&CmteCode=C232
 links:
+- url: https://capitol.texas.gov/Committees/MeetingsByCmte.aspx?Leg=88&Chamber=H&CmteCode=C232
 - url: https://capitol.texas.gov/Committees/MembershipCmte.aspx?LegSess=88R&CmteCode=C232
   note: homepage
 - url: https://capitol.texas.gov/Committees/MembershipCmteHist.aspx?LegSess=88R&CmteCode=C232
 - url: https://capitol.texas.gov/Reports/Report.aspx?ID=committee&LegSess=88R&Code=C232
-- url: https://capitol.texas.gov/Committees/MeetingsByCmte.aspx?Leg=88&Chamber=H&CmteCode=C232
+- url: https://www.house.texas.gov/committees/committee/?committee=C232
 members:
 - name: Brad Buckley
-  role: chair
+  role: 'chair '
   person_id: ocd-person/e3533cbc-f3ab-46bc-a591-c68657f82527
 - name: Barbara Gervin-Hawkins
-  role: vice chair
+  role: 'vice chair '
```

Comment:
Any chance we can just have this as a striped non-string? Noticed when `chair` or `vice chair` get added it's as 'chair ' & 'vice chair ', don't think it really matters but just curious

Reply:
Good catch

Reply:
Updated